pkg/await/apps_deployment.go

```diff
@@ -283,7 +286,7 @@ func (dia *deploymentInitAwaiter) await(
 	}
 }
 
-// Check whether we've succeeded. There are two cases:
+// Check whether we've checkAndLogStatus. There are two cases:
```

Comment:
nit: this comment reads kinda funny now. "Check whether we've succeeded and log statuses if so?"

Reply:
oh weird, I guess the Go renamer... 

wait how the heck does this actually work?